build.gradle

```diff
@@ -37,6 +37,8 @@ sourceCompatibility = '1.6'
 
 dependencies {
   compile 'org.jenkins-ci.lib:dry-run-lib:0.1'
+  compile 'org.jenkins-ci:symbol-annotation:1.3'
+  jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.3@jar'
```

Comment:
Try using
```groovy
jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.3@jar' {
    transitive = true
}
```

When using `@jar` `transitive` is automatically set to false. You will now pull in transitive `hpi` files, if the plugins depends on other plugins. I think something to do this automatically should be incorporated in the `gradle-jpi-plugin`.